src/Parser/FrostMage/Modules/Features/IcyVeinsDuration.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Module from 'Parser/Core/Module';
+
+class IcyVeinsDuration extends Module {
+  static dependencies = {
+		combatants: Combatants,
+	}
+	
+  on_initialized() {
+	const hasThermalVoid = this.combatants.selected.hasTalent(SPELLS.THERMAL_VOID_TALENT.id);
+	this.active = hasThermalVoid;
+  }
+
+  IcyVeinsCasts = 0;
```

Comment:
Variables should be initiated in the beginning of the class, preferably under `dependencies`.

Reply:
ok, will do.